pkg/authenticator/helmsecret_test.go

```diff
@@ -0,0 +1,47 @@
+package authenticator
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewHelmSecret(t *testing.T) {
+	t.Parallel()
+
+	t.Run("golden path", func(t *testing.T) {
```

Comment:
There's really no need for this test. It's not even a "golden path", because it doesn't actually _do anything_. While I can see the usefulness of such a test early on as you're writing the code, it's not worth making every dev run this test every time they run `make test`.

Reply:
Good call, this actually did more in the original implementation which is why it got stuck here.